apps/wallet/src/features/Swap/InputSwap/ToToken.tsx

```diff
@@ -0,0 +1,102 @@
+import { useCallback, useEffect } from 'react';
+import { StyleSheet } from 'react-native';
+import { Text, View } from '@walless/gui';
+import type { TokenDocument } from '@walless/store';
+import { throttle } from 'lodash';
+import { showError } from 'modals/Error';
+import type { JupiterToken } from 'utils/hooks';
+import { useSnapshot } from 'utils/hooks';
+import { getMappedMint, getSwapQuote } from 'utils/transaction';
+
+import type { SwapContext } from '../context';
+import { swapActions, swapContext } from '../context';
+
+import SelectButton from './SelectButton';
+
+const ToToken = () => {
+	const { fromToken, toToken, amount, swapQuote } = useSnapshot(swapContext)
+		.swap as SwapContext;
+
+	const handleSelectToToken = () => {
+		swapActions.openSelectToken('to');
+	};
+
+	const outAmount = swapQuote
+		? (parseInt(swapQuote.outAmount) * 1.0) / 10 ** (toToken?.decimals || 0)
+		: 0;
+
+	const updateSwapQuote = useCallback(
+		throttle(
```

Comment:
use `use-debounce` library for this, traditional `useCallback` + `throttle` did not work well in React Native, see [this line/file](https://github.cl/cocrafts/walless/blob/5dc1b4daa00788e0f90dc36ecac1c186fa58aa33/apps/mobile/src/utils/hooks/navigation.ts#L5) as reference.